src/table/Root.jsx

```diff
@@ -1,19 +1,16 @@
-import { createRoot } from 'react-dom/client';
 import React from 'react';
 import { StyleSheetManager } from 'styled-components';
 import { ThemeProvider } from '@mui/material/styles';
 import rtlPluginSc from 'stylis-plugin-rtl-sc';
 import TableWrapper from './components/TableWrapper';
 
-let reactRoot;
+let root;
```

Comment:
Do we need this? Can't we pass it to teardown as well? Might as well avoid global variables if we can

Reply:
No, we do not. Good idea. Done